src/main/java/com/google/devtools/build/lib/actions/SpawnResult.java

```diff
@@ -226,6 +227,8 @@ String getDetailMessage(
     private final Optional<Long> numInvoluntaryContextSwitches;
     private final boolean cacheHit;
     private final String failureMessage;
+    private final ActionInput inMemoryOutputFile;
```

Comment:
Is it possible for a spawn to have several output files?

Reply:
The `SpawnResult.getInMemoryOutput(ActionInput)` method supports multiple output files. However, so fare we only ever needed at most one. If someone needs more than one he's free to update this implementation to a list, but I think for now this is overkill.